config/webpack.config.base.js

```diff
@@ -110,7 +111,8 @@ module.exports = {
     new webpack.LoaderOptionsPlugin({
       minimize: true,
       debug: true
-    })
+    }),
+    new ExtractTextPlugin('styles.module.css')
```

Comment:
for this particular line, the parameter into `ExtractTextPlugin` is supposed to be the file name of the resulting bundled CSS file after webpack is done, however, it looks like what you currently have is the name of the existing CSS file. you'll want to use `app.css` or `styles.css` or some name that signifies that this is the final CSS bundle. additionally, if there will be multiple CSS files, you'll want to use the special notation for webpack to correctly name the result...

- [name] the name of the chunk
- [id] the number of the chunk
- [contenthash] a hash of the content of the extracted file

use this as a reference: https://github.com/webpack/extract-text-webpack-plugin

additionally, we want to configure webpack to put static assets inside the `build/static` directory. this is already the case for JS, `build/static/js`, so you'll want to do the same for CSS. this means you'll want to do something like the following:

```
// this is the equivalent of 'static/css/app.css'
new ExtractTextPlugin(`${PATHS.REL.STATIC_CSS}/app.css`)
```